.travis.yml

```diff
@@ -17,17 +17,17 @@ env:
 matrix:
   include:
     - php: 5.6
-      env: DB=MYSQL INSTALLER_VERSION=4.4.x-dev PHPCS_TEST=1 PHPUNIT_TEST=1
+      env: DB=MYSQL INSTALLER_VERSION=~4.4.0@dev PHPCS_TEST=1 PHPUNIT_TEST=1
     - php: 7.0
```

Comment:
Remove 5.6 + 7.0 from matrix

Add 7.4 + 8.0 (nightly)

Should only have a single `4.4.0@dev` entry in the matrix for php 7.1

Should have 4.5.x-dev, 4.6.x-dev (maybe 4.7.x-dev) and 4.x-dev

Reply:
then we'll have to upgrade to xenial and maybe then it's actually easier to go for shared config...

Reply:
done